ext/session/session.stub.php

```diff
@@ -43,35 +43,45 @@ function session_commit(): bool {}
  * @param callable|object $open
  * @param callable|bool $close
  */
-function session_set_save_handler($open, $close = UNKNOWN, callable $read = UNKNOWN, callable $write = UNKNOWN, callable $destroy = UNKNOWN, callable $gc = UNKNOWN, callable $create_sid = UNKNOWN, callable $validate_sid = UNKNOWN, callable $update_timestamp = UNKNOWN): bool {}
+function session_set_save_handler(
+    $open,
+    $close = UNKNOWN,
+    callable $read = UNKNOWN,
+    callable $write = UNKNOWN,
+    callable $destroy = UNKNOWN,
+    callable $gc = UNKNOWN,
+    callable $create_session_id = UNKNOWN,
+    callable $validate_session_id = UNKNOWN,
+    callable $update_timestamp = UNKNOWN
```

Comment:
These are called `create_sid()`, `validateId()` and `updateTimestamp()` in the SessionHandler interface -- yes, complete with inconsistent name casing. I was going to suggest that we should match those names here, but given the casing issue I'm not sure.

Reply:
No idea here either. I can revert to using the original names here until we decide it.